corsheaders/middleware.py

```diff
@@ -103,7 +103,7 @@ def process_response(self, request, response):
         Add the respective CORS headers
         """
         origin = request.META.get('HTTP_ORIGIN')
-        if request._cors_enabled and origin:
+        if getattr(request, '_cors_enabled', False) and origin:
```

Comment:
I think this is the wrong behaviour. I think if the attribute isn't there we should call `is_enabled` at this point, and add the correct headers!